src/decode-response/handle-rpc-error.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+var abi = require("augur-abi");
+var errors = require("../errors/codes");
+
+var handleRPCError = function (method, returns, response) {
+  var i, len, responseNumber;
+  if (response) {
```

Comment:
Are there any cases where non-batched RPC requests result in an array coming back from the node?  If not there are a number of `if array do X else do Y` conditionals that we could chop out of the code...